style-dictionary/visual-refresh/metadata/colors.ts

```diff
@@ -321,6 +321,16 @@ const metadata: StyleDictionary.MetadataIndex = {
     public: true,
     themeable: true,
   },
+  colorTextBreadcrumbCurrent: {
+    description: 'The text color that marks the breadcrumb item for the page the user is currently viewing.',
+    themeable: true,
+    public: true,
+  },
+  colorTextBreadcrumbIcon: {
```

Comment:
a bit strange that this has `text` in the name even though it's an icon color. But it also seems to be the way we name other icon colors too, so I guess 🤷